.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Bug
+about: To report a bug found in the bot's working.
```

Comment:
I'm not sure if the project is _only_ limited to the bot... Maybe there's some bug in an implementation of the API used rather than the bot?